data/interactive-defaults.ks

```diff
@@ -5,7 +5,7 @@ firstboot --enable
 
 %anaconda
 # Default password policies
-pwpolicy root --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy user --strict --minlen=8 --minquality=50 --nochanges --emptyok
-pwpolicy luks --strict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy root --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy user --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
+pwpolicy luks --notstrict --minlen=8 --minquality=50 --nochanges --emptyok
```

Comment:
Actually, it looks like the appropriate set of those was added back in the branch. I was looking at old information. Sorry for the noise.

Refs:
* https://github.com/rhinstaller/anaconda/commit/e8332fd43e9764962aa0f1fc85ac64ed2a2ca35d
* https://github.com/rhinstaller/anaconda/commit/9446c9fc77952e357770803865ff55d24e7d4b77
* https://github.com/rhinstaller/anaconda/commit/17755a403dcbfb6d0e98ea90526bf017bdbe14d7
* https://github.com/rhinstaller/anaconda/commit/82d67de0b255f3eb4044e91083cd8b907f81ac21
* https://github.com/rhinstaller/anaconda/commit/d79a74c0e69b1711b2f93284358aaacfd263c593
* https://github.com/rhinstaller/anaconda/commit/8f24eeaedd7691b6ebe119592e5bc09c1c42e181